packages/testcontainers/src/containers/SanityContainer/index.ts

```diff
@@ -0,0 +1,127 @@
+import { fetch, Response } from 'cross-fetch'
+import * as path from 'path'
+import { pack } from 'tar-fs'
+
+import { DockerContainer, hasImageLocally } from '../DockerContainer'
+import { MasterNodeRegTestContainer } from '../RegTestContainer/Masternode'
+
+const PROJECT_ROOT = path.resolve(__dirname, '../../../../../')
+
+/**
+ * Sanity Container
+ *
+ * We are running sanity containers seperately from the established unit tests as
+ * they are designed to test against a simulated prouduction deployment ensuring
+ * our containers are resolving incoming requests as expected.
+ *
+ * We do however introduce the container environment so that they can be run via
+ * unit tests to create uniformity in our approach to testing. Reducing the
+ * cognitive complexity of having to run multiple automated solutions to ensure
+ * release quality.
+ *
+ * This solution works by building a new docker image using our root Dockerfile
+ * and running instances on random ports to support parallel testing allowing for
+ * vertical scaling in our test environments.
+ */
+
+export abstract class SanityContainer extends DockerContainer {
```

Comment:
```suggestion
 */
export abstract class SanityContainer extends DockerContainer {
```

Reply:
This was originally deliberate but seeing its content description it makes sense to connect it directly to the class 👍🏻